lib/fastlane/plugin/wpmreleasetoolkit/actions/ios/ios_merge_strings_files.rb

```diff
@@ -0,0 +1,75 @@
+module Fastlane
+  module Actions
+    class IosMergeStringsFilesAction < Action
+      def self.run(params)
+        UI.message "Merging strings files: #{params[:paths].inspect}"
+
+        duplicates = Fastlane::Helper::Ios::L10nHelper.merge_strings(paths: params[:paths], output_path: params[:destination])
+        duplicates.each do |dup_key|
+          UI.important "Duplicate key found while merging the `.strings` files: `#{dup_key}`"
+        end
+        duplicates
+      end
+
+      #####################################################
+      # @!group Documentation
+      #####################################################
+
+      def self.description
+        'Merge multiple `.strings` files into one'
+      end
+
+      def self.details
+        <<~DETAILS
+          Merge multiple `.strings` files into one.
+
+          Especially useful to prepare a single `.strings` file merging strings from both `Localizable.strings` from
+          the app code — typically previously extracted from `ios_generate_strings_file_from_code` —
+          and string files like `InfoPlist.strings` — which values may not be generated from the codebase but
+          manually maintained by developers.
+
+          The action only supports merging files which are in the OpenStep (`"key" = "value";`) text format (which is
+          the most common format for `.strings` files, and the one generated by `genstrings`), but can handle the case
+          of different files using different encodings (UTF8 vs UTF16) and is able to detect and report duplicates.
+          It does not handle strings files in XML or binary-plist formats `.strings` files (which are valid but more rare)
```

Comment:
I tried looking for a webpage describing the OpenStep format but came up short. If you have one, it would be good to add it as a "see also" reference at the end of these details.

Reply:
The only links I could find (which are not really full docs about the format grammar but rather paragraphs in Apple's doc about some points about it) are:
 - [This page about strings file in the doc archive (so not even the current doc!)](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/LoadingResources/Strings/Strings.html#//apple_ref/doc/uid/10000051i-CH6-SW10)
 - On the same page, [this other paragraph near the end](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/LoadingResources/Strings/Strings.html#//apple_ref/doc/uid/10000051i-CH6-SW13)
 - [This paragraph in the wikipedia article about Property Lists](https://en.wikipedia.org/wiki/Property_list#NeXTSTEP), though this is more about the general NextStep PropertyList format (which supports more data types than just a `Dictionary<String,String>` root like `.strings` do) nor does it mention that `.strings` file use that format and can omit the surrounding `{ … }` of the root dictionary (and directly dump the key/value pairs that would otherwise typically be within those dictionary-scope-defining braces)… so not really complete either.

None of those are specifications nor even near being complete documentation and set of rules though. So not sure it's worth mentioning any of them here?

That's the whole issue and reason why I'm really reluctant in trying to generate strings files in this OpenStep/NextStep/ASCII format (well, except here because in this particular case it's basically just concatenating existing files already in that format, not creating entries ourselves and having to deal with all the escaping rules and all; and GlotPress only supports that format for `.strings` anyway, not binary nor XML), because I can't find an official specification for that format. It being deprecated and abandoned as an output format is probably the reason, but given it's still unofficially supported for reading and still used by Xcode for `.strings` and `.pbxproj` amongst other things, I'd still expect to find some legacy doc somewhere but still came up short.